src/kpro.erl

```diff
@@ -183,7 +183,7 @@
 -type count() :: non_neg_integer().
 -type wait() :: non_neg_integer().
 -type required_acks() :: -1..1 | all_isr | none | leader_only.
--type primitive() :: integer() | string() | binary() | atom().
+-type primitive() :: integer() | string() | binary() | [binary()] | atom().
```

Comment:
arguably array is no primitive, and there are integer arrays as well.
maybe change `filed_value()` to `:: primitive() | [primitive] | struct() | [struct]`

Reply:
done